scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1461,109 @@ def __getitem__(self, indexer):
                [ 0.57735027,  0.57735027, -0.57735027,  0.        ]])
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def match_vectors(cls, a, b, weights=None, normalized=False):
+        """Estimate the rotation to match two sets of vectors.
+
+        Find a rotation between frames A and B which best matches a set of unit
+        vectors `a` and `b` observed in these frames. The following loss
+        function is minimized to solve for the direction cosine matrix
+        :math:`C`:
+
+        .. math::
+
+            L(C) = \\frac{1}{2} \\sum_{i = 1}^{n} w_i \\lVert \\mathbf{a}_i -
+            C \\mathbf{b}_i \\rVert^2 ,
+
+        where :math:`w_i`'s are the `weights` corresponding to each vector.
+
+        The rotation is estimated using Markley's SVD method [1]_.
+
+        Parameters
+        ----------
+        a : array_like, shape (N, 3)
```

Comment:
The "usual"  way to write this seems to be `(N, 3), array_like`

Reply:
Don't want to be nasty, but we already used `array_like, shape (N, 3)` throughout whole `Rotation` class.  I suggest to use it here only because of this. (But somehow it stuck to my mind that this is the standard way.)

Reply:
If previously used, I guess yes better stick to it. In `linalg` module it is the other way around. 

IIRC this way it will try to render `shape`  as a type.